qutebrowser/browser/hints.py

```diff
@@ -748,6 +748,38 @@ def start(self, rapid=False, group=webelem.Group.all, target=Target.normal,
         self._context.tab.find_all_elements(selector, self._start_cb,
                                             only_visible=True)
 
+    def _get_visible_hints(self):
+        """Get elements which are currently visible.
+        """
```

Comment:
And I was wondering why is flake8 so much faster than pylint (which also didn't catch this by the way).